### DISCUSSION TITLE
Migration

### DISCUSSION THREAD

**Discussion by HPMSGEN8** [pymedusa/Medusa#10231] [Q&A] [answered]
Hi All,

How do i move my setup from Docker to  https://github.com/BenjV/SYNO-packages 

Also should i remove all NFO Files etc... and let it re-scan with TVMAZE ?

Thanks in advanced.

**Comment by BenjV** [comment]
Make a backup from within Medusa to a location (not the default location), where you can retrieve it later.
Install the correct medusa package (DSM6 or DSM 7)
Make sure that the "System Internal User" sc-medusa has read/write permissions on the location where you have stored the existing shows. 
Also make sure that the same "System Internal User" has read permission on the medusa backups
Restore the medusa backup.

It is useless to remove the NFO files etc...

**Comment by HPMSGEN8** [comment]
Hi @BenjV 

Thanks alot. One other Question if i want to change my TV Provider from TVDB to TVMAZE i take it i need to re-add each show to make this change. As i read somewhere that it is better to move to tvmaze due to the v2 API that needs to get paid for now. is this correct or not needed.

Thanks

**Comment by HPMSGEN8** [comment]
Also should you do an Update or Refresh on all your data to get everything to re-cache ?

**Reply by BenjV** [reply]
Yes the cache is not in the backup, so that info is gone.

**Reply by HPMSGEN8** [reply]
I mean do i need to do an Mass Update or Refresh. I take it Update is like an Force Update ?

**Reply by p0psicles** [reply]
If you restore from a backup, then you shouldn't have to update your shows.

**Reply by BenjV** [reply]
It not really needed because everything will be updated automatically in time, but a first you will be missing things like images and release schedules.
On every show page you can click on "Force Full Update" if you cannot wait or you can do a Mass Update for all Shows.

**Reply by BenjV** [reply]
@p0psicles 
Also the thumbnails?

**Reply by p0psicles** [reply]
ow sorry, no your right. The images from cache are not restored.

**Reply by HPMSGEN8** [reply]
Thanks alot as when i run watchlist i get an error stating the there is no tvdb number so it is not valid. So i take it that i need to do an Mass Update.

Thanks both :)

**Reply by p0psicles** [reply]
Sorry, i have no idea what you mean

**Comment by HPMSGEN8** [comment]
Hi @p0psicles 

There is an Kodi Add-On called WatchedList and this looks at all shows watched etc... If i giving me nfo issues as it cannot find the info. So what i want to know is that if I do an Update this will update all info for the show as well as the nfo correct or is this the Refresh Option.

Thanks

**Reply by BenjV** [reply]
Normally .nfo files are placed in the same folder as the video's so they should be still there.

**Reply by p0psicles** [reply]
Yeah but it's true that you'll keep your nfo files from your old indexer. So you might want to delete them and regenerate to get up2date nfo's

**Reply by p0psicles** [reply]
This PR whould fix that, https://github.com/pymedusa/Medusa/pull/10237

**Reply by HPMSGEN8** [reply]
Thanks alot @p0psicles  & @BenjV 

OH also i did what @BenjV  stated BUT my 720p WEB-DL & 1080p WEB-DL are showing downloaded SDTV  Or downloaded 720p HDTV / 1080p HDTV and it is trying to get all the files again. 

So i am not sure if this is an bug or if my Docker Version had some issues and now i need to fix it all.

**Reply by BenjV** [reply]
Just mark them as "archived" to stop the downloading again.

**Reply by HPMSGEN8** [reply]
Thanks.

**Comment by HPMSGEN8** [comment, marked as answer]
This can now get closed. Thanks alot :)
